napalm_logs/listener/tcp.py

```diff
@@ -29,11 +29,13 @@ class TCPListener(ListenerBase):
     '''
     TCP syslog listener class
     '''
-    def __init__(self, address, port, pipe):
+    def __init__(self, address, port, pipe, **kwargs):
         self.address = address
         self.port = port
         self.pipe = pipe
         self.__up = False
+        for key, value in kwargs.items():
```

Comment:
Are there any other args we expect for TCP? If not, we should simplify for the moment - if required later, we can add when necessary.